### PR TITLE
CLI testing with more complicated filters

### PR DESCRIPTION
**Pull request by arunverma87**
I also make changes as per suggested by you for CLI usage

### CODE REVIEW

File: app/plugins/eu.esdihumboldt.hale.app.transform.test/src/eu/esdihumboldt/hale/app/transform/test/ExecuteTest.groovy [hunk 1]
```diff
@@ -36,171 +36,339 @@ class ExecuteTest extends GroovyTestCase {
 	private static final String HYDRO_PROJECT = "platform:/plugin/$PLUGIN_NAME/projects/hydro/hydro-basic.halez"
 	private static final String HYDRO_DATA = "platform:/plugin/$PLUGIN_NAME/projects/hydro/hydro-source.gml.gz"
 
+	private static final String MULTI_TYPE_PROJECT = "platform:/plugin/$PLUGIN_NAME/projects/multitype/multi-type.halez"
```
Comment (stempler) [inline app/plugins/eu.esdihumboldt.hale.app.transform.test/src/eu/esdihumboldt/hale/app/transform/test/ExecuteTest.groovy]: In version control you should always store hale projects as `.halex`. The `.halez` format is actually a Zip file containing the project and any local resources.

File: app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy [hunk 2]
```diff
@@ -150,27 +150,36 @@ $baseCommand
...
-  If instance matches -exclude-type filter, then it will be excluded from the transformation, irrespective of trueness of other filters.
-
-  You can also specify all the parameters in file using -args-file parameter. File content should follow specific rule. e.g. 
-  -param1
-  value
-  -param2
-  value
-  ...
-  ...""".trim()
+  Sources
```
Comment (stempler) [inline app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy]: So you included my changes from #86 in your pull request? Not really what was intended, but you can leave it in here.

**Comment by stempler** [comment]
Please change the hale project format and resolve the conflicts for the pull request to be merged. If you have questions, please don't hesitate to ask.

File: app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy (hunk 2 above)
Comment (arunverma87) [inline app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy]: I did it manually, I was reading your changes and did it simultaneously on ma side.

**Pull request merged**